cloud-controller-manager/do/loadbalancers.go

```diff
@@ -40,6 +40,18 @@ const (
 	// used to enable fast retrievals of load-balancers from the API by UUID.
 	annoDOLoadBalancerID = "kubernetes.digitalocean.com/load-balancer-id"
 
+	// annDOLoadBalancerName is the annotation used to specify a name of the load
+	// balancer that is going to be created by the controller.
```

Comment:
Sure, I will improve the comment, and extend docs. I think it might be also useful to mention in docs that reconciliation errors can be found in the load balancer events stream.

Reply:
>I think it might be also useful to mention in docs that reconciliation errors can be found in the load balancer events stream.

That's an excellent suggestion. Let's capture that in a separate PR.

Reply:
Done :arrow_right: https://github.com/digitalocean/digitalocean-cloud-controller-manager/pull/310

Reply:
Not be misunderstood: we should update the comment here in this PR that LB updates are covered too.